docs/source/dataframe-joins.rst

```diff
@@ -58,8 +63,29 @@ maintains that index, like Parquet.
 
 .. code-block:: python
 
-    left = left.set_index('id').persist()
+    from dask.datasets import timeseries
+    from dask.dataframe import read_parquet
+
+    left = timeseries(dtypes={"foo": int}, freq="1s")
```

Comment:
I don't think `freq` is adding to the example here since it is always the same. 

Also I have a preference for meaningful names for the columns - something like `temperature` or `name` or `amount` or `lat` (same for `bar` and `baz` below). But there are tons of examples that use foo and bar in the dask docs, so I'm ok if you'd rather stick with that convention.

Reply:
I might do that, but then I think I will have to create some data that matches the column names. This is possible by passing keyword arguments like:

```
# generate sales column (poisson with lambda=1)
left = dask.datasets.timeseries(dtypes={"sales": int}, sales_lam=1)
```

I think this might create some overhead while people are just trying to learn about joins. Feels more like a tutorial on datasets API. Note that we need to pass `sales_lam` kwarg, which is then passed to `make_int` function. It's your call though.

Reply:
I'm fine leaving the column names as they are.